src/components/Theme/style.css

```diff
@@ -67,6 +63,11 @@
     /* A full screen step will be a video/stream step, with a darker background */
     background-image: url('assets/powered-by-onfido-light.svg');
   }
+
+  /* hide the footer if view height is too small */
+  @media (max-height: 420px) {
+    z-index: -1;
```

Comment:
Interesting way of hiding something...

Reply:
any particular reason for the `z-index`? could we not used visibility: hidden for example?

Reply:
This was already in the code and it was removed in the previous PR. I don't remember why we used z-index instead of `visibility: hidden`/`display: none`. I will replace it with `visibility: hidden`, meanwhile I'm also looking into why this is happening.

Reply:
I removed this hack and added `overflow: scroll` to `.content`

Reply:
`visibility: hidden` is different from `display: none`:

- `visibility: hidden` will hide the content but would still take place within the rendered area
- `display: none` will hide the content and also not going to take any place in the area

Whatever works best for this case.